source/stylesheets/refills/_comment.scss

```diff
@@ -56,7 +48,7 @@
 
     p {
       line-height: 1.5em;
-      margin-bottom: 0.5em;
+      margin: 0 0 0.5em 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 0 0.5em` instead of `0 0 0.5em 0`